test/gtest/uct/uct_test.cc

```diff
@@ -207,6 +207,9 @@ std::vector<const resource*> uct_test::enum_resources(const std::string& tl_name
 }
 
 void uct_test::init() {
+    if (RUNNING_ON_VALGRIND && (GetParam()->tl_name == "cm")) {
+        modify_config("ASYNC_MODE", "thread_mutex");
```

Comment:
maybe this should be done automatically in UCP/UCS async?
what if we run MPI job with valgrind?

Reply:
Yes, this is removed in next commits.